Rework a bit the April content for step 3 (#620)
akeneo/pim-helpcenter: content/updates/2020-04/6-rules-engine-new-actions-and-improvements.md

# Rules engine new actions and improvements

> To extend the rules engine capabilities, we added two new actions, the possibility to name your rules and we also improved the rules format for associations.

Author: ```suggestion
Discover the latest enhancements on the rules engine: we added two new actions, the possibility to name your rules and we also improved the rules format for associations.
```

> To extend the rules engine capabilities, we added two new actions, as well as the possibility to name your rules and improved the rules format for associations.

Author: ```suggestion
To extend the rules engine capabilities, we added two new actions, the possibility to name your rules and we also improved the rules format for associations.
```
